bot/exts/moderation/slowmode.py

```diff
@@ -16,7 +16,7 @@
 
 COMMONLY_SLOWMODED_CHANNELS = {
     Channels.python_general: "python_general",
-    Channels.discord_py: "discordpy",
+    Channels.discord_bots: "discordpy",
```

Comment:
We should change the channel name here as well.
```suggestion
    Channels.discord_bots: "discord_bots",
```

Reply:
Should probably have a comment why

Reply:
Originally, it was to avoid messing with stats too much. Just asked @jb3 on Discord and he says we should be good to rename the stat too as it's not critical or checked much, so I'm committing this.

Reply:
👍